webpack/webpack.common.js

```diff
@@ -57,7 +37,10 @@ module.exports = {
     ],
   },
   resolve: {
-    modules: [path.resolve(__dirname, "../webcompat/static/js/vendor")],
+    modules: [
+      path.resolve(__dirname, "../webcompat/static/js/vendor"),
+      "node_modules",
```

Comment:
This just means you can look inside `node_modules` for a module to include in the webpack bundle, correct?

Reply:
yeah, so we could just import a package from node_modules, without specifying the full path.
it'll be just `import "package_name"`